moveit_core/utils/include/moveit/utils/robot_model_test_utils.h

```diff
@@ -54,23 +54,26 @@ namespace moveit
 namespace core
 {
 /** \brief Loads a robot from moveit_resources.
- * \param[in] robot_name The name of the robot package in moveit_resources to load.
- *            For example, "panda_description", or "fanuc_description".
+ * \param[in] robot_name The name of the robot in moveit_resources to load.
+ *            This should be the prefix to many of the robot packages.
+ *            For example, "pr2", "panda", or "fanuc".
  * \returns a RobotModel constructed from robot_name's URDF and SRDF.
  */
 moveit::core::RobotModelPtr loadTestingRobotModel(const std::string& robot_name);
 
 /** \brief Loads a URDF Model Interface from moveit_resources.
- * \param[in] robot_name The name of the robot package in moveit_resources to load.
- *            For example, "panda_description", or "fanuc_description".
+ * \param[in] robot_name The name of the robot in moveit_resources to load.
+ *            This should be the prefix to many of the robot packages.
+ *            For example, "panda", or "fanuc".
```

Comment:
no "pr2" here?

Reply:
Quick mistake, thanks for pointing that out.